Gems/Atom/RHI/Code/Include/Atom/RHI.Reflect/PhysicalDeviceDescriptor.h

```diff
@@ -22,14 +22,17 @@ namespace AZ
     {
         //! A list of popular vendor Ids. 
         AZ_ENUM_CLASS_WITH_UNDERLYING_TYPE(VendorId, uint32_t,
-            (Unknown,  0),
-            (Intel,    0x8086),
-            (nVidia,   0x10de),
-            (AMD,      0x1002),
-            (Qualcomm, 0x5143),
-            (Samsung,  0x1099),
-            (ARM,      0x13B5),
-            (Warp,     0x1414)
+            (Unknown,       0),
+            (Intel,         0x8086),
+            (nVidia,        0x10de),
+            (AMD,           0x1002),
+            (Qualcomm,      0x5143),
+            (Samsung,       0x1099),
+            (ARM,           0x13B5),
+            (Warp,          0x1414),
+            (AppleA13GPU,   0x2c4),
+            (AppleM1Max,    0x983)
```

Comment:
seems odd to switch to per chip id's for apple, vs per vendor for others.

Reply:
I don't see any API that can provide vendor id.